azure-webapp-maven-plugin/README.md

```diff
@@ -242,7 +244,7 @@ Property | Required | Description | Version
     Supported `<os>` values are *Windows*, *Linux* and *Docker*.
     Only the `jre8` is supported for `<javaVersion>` for Web App on Linux.
     If the `<webContainer>` is not configured and the `<os>` is windows, tomcat 8.5 will be used as default value.
-    If the `<os>` is linux, the web app will use the JavaSE as runtime.
+    But if the `<os>` is linux, the web app will use the JavaSE as runtime.
```

Comment:
Shall we use a `,` instead of `.` here? for i think they should be in the same sentence.